### PR TITLE
Cedar - Kristin Lee, Maria Obregon

### PR DESCRIPTION
**Pull request by mobregong**

### CODE REVIEW

File: adagrams/commented_game.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    }
+
+def draw_letters():
+    # 3 tests
+    # no parameters
+    # LETTER_POOL is a dictionary
+    # Return an array of ten strings where each contain one letter 
+    # and are randomly drawn from LETTER_POOL
+    # Idea 1: 
+    # Before pulling letters, multiple the keys by the values (frequency) and append to a list
```
Comment (kaidamasaki) [inline adagrams/commented_game.py:49]: It's helpful if you include the comments in your main file instead of turning in two copies.  

Generally the only comments you are expected to remove are ones with commented out code.

File: adagrams/commented_game.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    # if letter in dict[key]:
+        # pointage += key
+# if length of word is range 7 to 11, then add 8 points
+
+    
+    point_count = 0
+    for letter in word.upper():
+        for key, value in SCORE_CHART_DICT.items():
+            if letter in SCORE_CHART_DICT[key]:
+                point_count += key
```
Comment (kaidamasaki) [inline adagrams/commented_game.py:118]: You could write this a little more clearly if you had `SCORE_CHART_DICT` go from letters to points.

It is relatively simple to build a reversed version of the dictionary (that you can store as a constant):

```python
LETTERS_TO_SCORES = {}

for score, letters in SCORE_CHART_DICT:
    for letter in letters:
        LETTERS_TO_SCORES[letter] = score
```

Then you can use `LETTERS_TO_SCORES` like this:

```suggestion
        if letter in LETTERS_TO_SCORES:
            point_count += LETTERS_TO_SCORES[letter]
```

File: adagrams/commented_game.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    # elif len(max_word[0]) == 10 or len(max_word[1]) == 10:
+    elif len(max_word[0]) == 10:
+        winning_word = max_word[0]
+    elif len(max_word[1]) == 10:
+        winning_word = max_word[1]
+    # if all are less than 10 letters, shortest wins
+    elif len(max_word[0]) < len(max_word[1]):
+        winning_word = max_word[0]
+    else:
+        winning_word = max_word[1]
```
Comment (kaidamasaki) [inline adagrams/commented_game.py:169]: What happens if we have a tie between more than two words?

(Hint: this code doesn't handle it properly. :-/)

**Review by kaidamasaki — COMMENTED** [review]
Great job!  

There were a couple of little things and in the future you should try to think about additional input data than just what we provided in the tests.

We're only human and will forget to test for edge cases sometimes.  :wink:
